README.md

```diff
@@ -14,30 +28,99 @@ To install multiple versions of Node.js, you may find it easier to use a node ve
 * [nvm](https://github.com/creationix/nvm)
 * [n](https://github.com/tj/n)
 
-### Installation
+
+## Installation
+
+Clone this repo and install its dependencies...
 
 ```
 $ npm install
 ```
 
-### Usage
+You'll then need to duplicate the `config.sample.json` file as `config.json` and update its contents...
```

Comment:
They have to get the config values themselves. We can't say where they can get them from as it could be anywhere.